docs/src/main/asciidoc/extension-authors-guide.adoc

```diff
@@ -1233,6 +1233,130 @@ HealthBuildItem addHealthCheck(AgroalBuildTimeConfig agroalBuildTimeConfig) {
 }
 ----
 
+=== Customizing Json handling from an extension
+It is often needed for an extension to register serializer/deserializer for specific types that the extension provides.
+
+For this, both JsonB and Jackson extension provide a way to register serializer/deserializer from within an
+extension deployment module.
+
+Keep in minds that not everybody will need json, so you need to make it optional.
+It is strongly advised to provide customization for both JsonB and Jackson.
+
+==== Customizing JsonB
+First, add an *optional* dependency to `quarkus-jsonb` on your extension's runtime module.
+
+[source%nowrap,xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-jsonb</artifactId>
+  <optional>true</optional>
+</dependency>
+----
+
+Then create a serializer or a deserializer for JsonB, example is from the `mongodb-panache` extension.
```

Comment:
`or` -> `and/or`

Reply:
`example is from the` -> `an example of which can be seen in the`

Reply:
done